src/neuvector/values/values.yaml

```diff
@@ -13,32 +13,19 @@ controller:
 
   apisvc:
     type: ClusterIP
-  secret:
-    enabled: true
+  configmap:
+    enabled: false
```

Comment:
This should be true for the configmap I think? To make sure we load in our telemetry/scan config?